ensemble/volren/volume_data_filter.py

```diff
@@ -0,0 +1,11 @@
+class VolumeFilter():
+    """ This is the default volume data filter. 
+        Subclass this filter to transform the volume data.
+    """
+    name = 'Default'
```

Comment:
As it is defined, this is a class attribute, which means that all the instances will have the same name. However, it's possible that we will have parametric families of filters, in which case we will want to assign different names to different instances. Please make this into a `Str` trait.